api/v1alpha1/spacerequest_types.go

```diff
@@ -0,0 +1,86 @@
+package v1alpha1
+
+import (
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+// SpaceRequestSpec defines the desired state of Space
+// +k8s:openapi-gen=true
+type SpaceRequestSpec struct {
+	// TierName is a required property introduced to retain the name of the tier
+	// for which this Space is provisioned.
+	TierName string `json:"tierName"`
+
+	// TargetClusterLabelKeys one or more label keys that define a set of clusters
+	// where the Space can be provisioned.
+	// +optional
```

Comment:
Since it's been changed to a list, then we should also define the type, right?
```suggestion
	// +optional
	// +listType=atomic
```

Reply:
good catch, thanks !